SU-SOE/stanford_soe_helper
# 7.x 2.x collection node

# READY FOR REVIEW

# Summary
- This incorporates CSS for the Collection node display page as requested by Kerri with some clean up.

# Needed By (3/28/2018)
- Sprint end

# Criticality
- Fixes broken stuff
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch
- Flush cache
- Navigate to a collection node to verify


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2740
## Related PRs

## More Information

## Folks to notify
@minorwm


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
